ui/ts/pages/log.ts

```diff
@@ -94,16 +94,33 @@ module AdminViews {
           entries.refresh();
         }
 
-        constructor(nodeId?: string) {
-          entries = new Models.Log.Entries(nodeId);
+        constructor() {
+          entries = new Models.Log.Entries();
+          if (m.route.param("node_id")) {
```

Comment:
I don't think you need the 'if' statements for the string params, `m.route.param()` returns `undefined` if the param is not present.

If you really need null, you could just do entries.node(m.route.param("node_id") || null)